### PR TITLE
Healthcare 13732 integration with api

### PR DESCRIPTION
**Pull request by mdewey**
## Description
Updated aspects related to calling the API for data. 

- Updated when to call the real API 
- Updated to a agreed upon data structure for appointments

## Testing done
- unit and e2e tests were updated

## Screenshots
- N/A

## Acceptance criteria
- [x] Pulls the API in all by cypress testing. 

## Definition of done
- [x] [Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/13732)

### CODE REVIEW

File: src/applications/healthcare/questionnaire/api/appointment-data.json [hunk 1]
```diff
@@ -1,57 +1,42 @@
...
+				"status": "defgh",
+				"patients": [{
+					"name": "Foo Bar"
+				}],
+				"providers": [{
+					"name": "The Provider"
+				}]
+			}]
+		}
+	}
```
Comment (dillo) [inline src/applications/healthcare/questionnaire/api/appointment-data.json:41]: Is this still needed? Are we pulling the mock appointment from the vets-api yet?

**Review by dillo — APPROVED** [review]
Good to merge after the answer to my comment is addressed 👍

File: src/applications/healthcare/questionnaire/api/appointment-data.json (hunk 1 above)
Comment (mdewey) [reply src/applications/healthcare/questionnaire/api/appointment-data.json:41]: Yes, for e2e and unit testing environments.

Comment (mdewey) [reply src/applications/healthcare/questionnaire/api/appointment-data.json:41]: To clarify, yes its needed for the testing, and yes we are pulling from the vets-api

**Pull request merged**